integration/ft_info_parser.py

```diff
@@ -298,6 +298,41 @@ def with_offsets(self) -> Optional[bool]:
         offsets = self.parsed_data.get("with_offsets")
         return bool(offsets) if offsets is not None else None
 
+    @property
+    def num_unique_terms(self) -> Optional[int]:
+        """Get the number of unique terms in the text index."""
+        return self.parsed_data.get("num_unique_terms")
+
+    @property
+    def num_total_terms(self) -> Optional[int]:
+        """Get the total frequency of all terms across all documents."""
+        return self.parsed_data.get("num_total_terms")
+
+    @property
+    def posting_sz_bytes(self) -> Optional[int]:
+        """Get the memory used by posting lists (inverted index data) in bytes."""
+        return self.parsed_data.get("posting_sz_bytes")
+
+    @property
+    def position_sz_bytes(self) -> Optional[int]:
+        """Get the memory used by position information for phrase queries in bytes."""
```

Comment:
Might be good to mention that it's a subset of the postings object memory which I assume is also already included in `posting_sz_bytes`. It's not immediately clear from these comments how the memory is divided or double-counted across the different pieces here